test/fixtures/_partial_with_helpers.html.erb

```diff
@@ -0,0 +1,13 @@
+<%
+  partial.helpers do
+    def upcase(content)
+      content.upcase
+    end
+
+    def squish(content)
+      content.gsub(/\s+/, " ")
+    end
+  end
+%>
+
+<p><%= upcase yield %></p>
```

Comment:
I don't understand why we can call `upcase` without doing `partial.upcase`?

Reply:
Is this benefiting from a misuse of a `Section` definition?

https://github.com/bullet-train-co/nice_partials/blob/main/lib/nice_partials/partial.rb#L107

Reply:
Hm, I'm not quite sure what's going on here, I'll try to look locally.